release_notes_generator/model/record.py

```diff
@@ -154,6 +154,11 @@ def get_rls_notes(self, detection_pattern: str, line_marks: Optional[list[str]]
             inside_release_notes = False
 
             for line in body_lines:
+                stripped_line = line.strip()
+                if len(stripped_line) == 0:
+                    # skip empty lines as they are not relevant
+                    continue
+
```

Comment:
I don't mind it like this, but we can simplify that 
```suggestion
if not line.strip():
       continue  # skip empty lines as they are not relevant
```

Reply:
Nice catch. Fixed.